automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -588,10 +588,11 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
               logger.warn(s"Could not obtain localization log files for cluster ${cluster.projectNameString}: ${e.getMessage}")
           }
 
+          //TODO:: the code below messes up the test somehow, figure out why that happens and fix.
```

Comment:
if there's another ticket for this, it might be worth referencing it here for more context

Reply:
https://github.com/DataBiosphere/leonardo/issues/643